{"name": "test_2", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 2", "trace": "def test_2():\n        print(\"test_2\")\n>       assert 1 == 2\nE       assert 1 == 2\n\n../test_case/test_case_01.py:8: AssertionError"}, "attachments": [{"name": "stdout", "source": "412d82e1-a611-4b74-96a5-9809a96e3a6b-attachment.txt", "type": "text/plain"}], "start": 1587534714953, "stop": 1587534714953, "uuid": "fb4cfd07-cf2b-4892-8595-aa4d7a1a3ee3", "historyId": "b6214ae89660c97e3a598e01dae1ba06", "testCaseId": "cc6896423e23cb37cfe55037c3b2ef64", "fullName": "test_case.test_case_01#test_2", "labels": [{"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_case_01"}, {"name": "host", "value": "CreamK"}, {"name": "thread", "value": "42948-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_case_01"}]}